src/vs/workbench/browser/web.main.ts

```diff
@@ -213,11 +213,14 @@ class BrowserMain extends Disposable {
 		serviceCollection.set(IUserDataInitializationService, userDataInitializationService);
 
 		if (await userDataInitializationService.requiresInitialization()) {
+			mark('willInitRequiredUserData');
```

Comment:
@sandy081 if you want to see this in the perf editor I think you will have to add this into the `IStartupMetrics` interface.

Reply:
Ah I did not know about that. I do not want to add more changes for recovery, hence, I will remove these perf markups and do them correctly in main branch.